### PR TITLE
Address several XR issues

### PR DESCRIPTION
**Pull request by m4gr3d**
This PR contains four commits with the following updates:

- Address rebasing merge issues
- Add a splash screen on Meta devices
- Add scrolling capability to the controller pointer
- Fix the issue causing tracking to break in the XR Editor

### CODE REVIEW

**Review by BastiaanOlij — APPROVED** [review]
lgtm

**Comment by BastiaanOlij** [comment]
Not sure about the linux failure, it's trying to add OpenXRInterface as a global, which it's not. There is no guarantee of this existing. Did we accidentally register it as a singleton somewhere?

File: modules/openxr/register_types.cpp [hunk 1]
```diff
@@ -139,6 +139,7 @@ void initialize_openxr_module(ModuleInitializationLevel p_level) {
 		XRServer *xr_server = XRServer::get_singleton();
 		if (xr_server) {
 			openxr_interface.instantiate();
+			Engine::get_singleton()->add_singleton(Engine::Singleton("OpenXRInterface", openxr_interface.ptr(), "OpenXRInterface"));
```
Comment (BastiaanOlij) [inline modules/openxr/register_types.cpp]: Ah, here it's added... Let's not do this. Registered singletons need to be guaranteed to exist, OpenXRInterface only exists when used.

If you need access to the interface from GDScript simply access `XRServer.primary_interface`

Makes the code more portable if we ever want the editor to work with other interfaces in the future as well.

Comment (m4gr3d) [reply modules/openxr/register_types.cpp]: Fixed. This was done to address a bug caused by the creation of two `OpenXRInterface` instance. I used a different approach to resolve the bug without making `OpenXRInterface` a registered singleton.

Comment (BastiaanOlij) [reply modules/openxr/register_types.cpp]: Ah easy :)

**Comment by BastiaanOlij** [comment]
I don't know what the build issue on Windows is about... ?

**Comment by m4gr3d** [comment]
> I don't know what the build issue on Windows is about... ?

There are no changes in the PR that affect / touch Windows' specific files; could it be an issue with the rebase?

**Comment by BastiaanOlij** [comment]
> > I don't know what the build issue on Windows is about... ?
> 
> There are no changes in the PR that affect / touch Windows' specific files; could it be an issue with the rebase?

Could be, I might just merge this and then squash and rebase the main PR.

**Comment by BastiaanOlij** [comment]
Ok we probably do need to undo your changes to the init code because I don't think that is right. This will likely break the Vulkan implementation as things are happening in the wrong order now. I need to look into this further.

**Comment by BastiaanOlij** [comment]
Ok, having a closer look at those changes, I think this is indeed the right way to do it. Might solve some other issues too.

**Pull request merged**